processor/resourcedetectionprocessor/internal/gcp/metadata.go

```diff
@@ -4,54 +4,61 @@
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//     http://www.apache.org/licenses/LICENSE-2.0
+//       http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Is this change intentional?

Reply:
Not sure why or how it got changed.. reverted.